cookbook/how_to_pass_request_parameter_to_grid.rst

```diff
@@ -94,74 +194,8 @@ Register this listener in container:
             tags:
                 - { name: kernel.event_listener, event: oro_datagrid.datagrid.build.after.acme-tasks-grid, method: onBuildAfter }
 
-Now we need to pass parameter with name "contactId" to our grid.
-The controller receives a contact entity and pass it to the view:
-
-.. code-block:: php
-    :linenos:
-
-    <?php
-        // src/Acme/Bundle/TaskBundle/Controller/TaskController.php
-        namespace Acme\Bundle\TaskBundle\EventListener;
-
-        use Sensio\Bundle\FrameworkExtraBundle\Configuration\Route;
-        use Sensio\Bundle\FrameworkExtraBundle\Configuration\Template;
-
-        use Symfony\Bundle\FrameworkBundle\Controller\Controller;
-
-        use OroCRM\Bundle\ContactBundle\Entity\Contact;
-
-        class TaskController extends Controller
-        {
-            // ...
-
-            /**
-             * @Route("/contact/{id}/tasks", name="acme_task_contact_tasks", requirements={"id"="\d+"})
-             * @Template
-             */
-            public function contactTasksAction(Contact $contact)
-            {
-                return array('contact' => $contact);
-            }
-
-            // ...
-        }
-
-The view passes parameter "contactId" to the grid, it will be used in the listener:
-
-.. code-block:: html+jinja
-    :linenos:
-
-    {# src/Acme/Bundle/TaskBundle/Resources/views/Task/contactsTask.html.twig #}
-    {% import 'OroDataGridBundle::macros.html.twig' as dataGrid %}
-
-    <div class="widget-content">
-        {{ dataGrid.renderGrid('acme-tasks-grid', {contactId: contact.id}) }}
-    </div>
-
-
-Solution 2. Use existing listener
----------------------------------
-
-Instead of writing your custom listener you can use existing class
-(Oro\Bundle\DataGridBundle\EventListener\BaseOrmRelationDatagridListener) that can be referenced in services configuration
-via parameter "oro_datagrid.event_listener.base_orm_relation.class":
-
-.. code-block:: yaml
-    :linenos:
-
-    # src/Acme/Bundle/TaskBundle/Resources/config/services.yml
-    services:
-        acme_task.event_listener.acme_tasks_grid_parameter_listener:
-            class: %oro_datagrid.event_listener.base_orm_relation.class%
-            arguments:
-                - contactId
-                - false
-            tags:
-                - { name: kernel.event_listener, event: oro_datagrid.datagrid.build.after.acme-tasks-grid, method: onBuildAfter }
-
-This way the listener is reused and you don't need to write yours, but you still need to pass parameter "contactId"
-to the grid (see example with passing parameters in the grid from Twig template).
+Remember that you still need to pass the "relatedContactId" parameter to the grid to the grid from a Twig template
```

Comment:
I would enclose the parameter name with double backticks for proper highlighting.